### PR TITLE
US Data Edit works

### PR DESCRIPTION
**Pull request by vtruo009**
US Data editing function works; tested by putting edit function in USData.js and calling it by passing in hardcoded values, and function returns true after editing.

### CODE REVIEW

File: src/Server/routes/USData.js [hunk 1]
```diff
@@ -75,4 +75,10 @@ router.post('delete/', (req, res) => {
 	})
 });
 
+router.post('edit/', (req, res) => {
```
Comment (johan1505) [inline src/Server/routes/USData.js]: change 'edit/' to '/update'

Comment (vtruo009) [reply src/Server/routes/USData.js]: changed

Comment (vtruo009) [reply src/Server/routes/USData.js]: nvm not changed

**Comment by johan1505** [comment]
Merge conflicts

**Pull request merged**